examples/atlas-organization/organization-step-1/Readme.md

```diff
@@ -0,0 +1,76 @@
+# Example - A basic example to create and Orgnization with MongoDB Atlas and Terraform
```

Comment:
I think what's missing is a README in atlas-organization folder (directly above) explaining how Step 1 and Step 2 directories relate to one another. Confusing looking at this for the very first time.